<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MWindow</class>
 <widget class="QMainWindow" name="MWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>557</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>920</width>
    <height>557</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>920</width>
    <height>557</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Навигация</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QRadioButton" name="floor1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>83</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1 этаж</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">floors</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="floor2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>83</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>2 этаж</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">floors</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="floor3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>83</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>3 этаж</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">floors</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="floor4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>180</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>4 этаж </string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">floors</string>
    </attribute>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Выберите этаж</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showPicture">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Показать план этажа</string>
    </property>
   </widget>
   <widget class="QLabel" name="for_picture">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>511</width>
      <height>401</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="findClass">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Найти кабинет</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ClassNumber">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Введите номер кабинета:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showClass">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>390</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Показать его на плане</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>920</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="floors"/>
 </buttongroups>
</ui>
